javascript/index.js

```diff
@@ -14,52 +15,122 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  getMinutes(); //return minutes
+  getSeconds()// return seconds
+  twoDigitsNumber(minutes)
+  twoDigitsNumber(seconds)
+
+  minDec.innerHTML = minutes[0]
```

Comment:
you are already changing the DOM elements in the printMinutes and printSeconds functions. So instead of changing the same DOM elements in here, just call those functions.